pages/disability/file-disability-claim-form-21-526ez.md

```diff
@@ -1,17 +0,0 @@
----
-title: Apply for disability benefits
-entryname: 526EZ-all-claims
-layout: page-react.html
-description: Learn how to apply online for disability compensation.
-hideFromSidebar: true
-vagovprod: true
----
-
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/">Home</a></li>
-    <li><a href="/disability/">Disability Benefits</a></li>
-    <li><a aria-current="page" href="/disability/file-disability-claim-form-21-526ez/">File for Disability Compensation</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-4cccce8a6a3925c798cf79529fec14efR5. Fixed a breadcrumb in 77f616c19c88c40383bdde44c940bf3671786af8.